### PR TITLE
Create helm_chart_gateway_service_annotations.yaml

### PR DESCRIPTION
**Pull request by 4c74356b41**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[x ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: releasenotes/notes/helm_chart_gateway_service_annotations.yaml
Comment (Kmoneal) [inline releasenotes/notes/helm_chart_gateway_service_annotations.yaml]: `bug-fix` instead of `feature`

File: releasenotes/notes/helm_chart_gateway_service_annotations.yaml
Comment (Kmoneal) [inline releasenotes/notes/helm_chart_gateway_service_annotations.yaml]: go ahead and delete empty fields

Comment (Kmoneal) [inline releasenotes/notes/helm_chart_gateway_service_annotations.yaml]: go ahead and delete empty fields

**Comment by Kmoneal** [comment]
Along with the comments, I recommend naming it after the pr #

**Comment by 4c74356b41** [comment]
@Kmoneal done. can you, please, check again.

**Review by Kmoneal — APPROVED** [review]
@4c74356b41 lgtm

**Comment by 4c74356b41** [comment]
@Kmoneal I cant merge this PR. only people with write access can merge it.
image

**Comment by Kmoneal** [comment]
We'll just have to add the release note in manually for the next release then. Feel free to close this out and I will approve the cherry-pick. Sorry this was a bit of a process.

**Pull request closed without merging**
